bin/gpuarray-cache

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import os
+
+def clean(max_size):
+    content = []
+    for root, dirs, files in os.walk(os.environ.get('GPUARRAY_CACHE',
+                                                    '~/.gpuarray/cache/')):
```

Comment:
Why set a default here when if GPUARRAY_CACHE isn't set, we don't have a default?

If it work well on Linux with NFS, I would enable it by default on Linux. Until it is more tested on mac and windows, we should wait for those platform.